templates/snmpd.service-Debian.erb

```diff
@@ -0,0 +1,18 @@
+###
```

Comment:
Can you provide the template as an epp instead of an erb template?

Reply:
II do'nt know EPP yet so I can just rename the file

Reply:
Moreover this will break : 
`$template_snmpd_service       = "snmp/snmpd.service-${facts['os']['family']}.erb"`